### PR TITLE
Update zh_CN.lang

### PR DESCRIPTION
**Pull request by forestbat**

### CODE REVIEW

File: src/main/resources/assets/thermalfoundation/lang/zh_CN.lang [hunk 1]
```diff
@@ -452,11 +509,16 @@ tile.thermalfoundation.glass.tin.name=硬化锡玻璃
 
 tile.thermalfoundation.ore.aluminum.name=铝矿石
 tile.thermalfoundation.ore.copper.name=铜矿石
+tile.thermalfoundation.ore.crude_oil_gravel.name=油页岩
+tile.thermalfoundation.ore.crude_oil_sand.name=油砂
+tile.thermalfoundation.ore.ender.name=谐振末影矿石
+tile.thermalfoundation.ore.glowstone.name=充能萤石地狱岩
```
Comment (3TUSK) [inline src/main/resources/assets/thermalfoundation/lang/zh_CN.lang]: 荧石。

File: src/main/resources/assets/thermalfoundation/lang/zh_CN.lang [hunk 2]
```diff
@@ -385,6 +430,18 @@ item.thermalfoundation.tool.silverShovel.name=银锹
...
 
+item.thermalfoundation.tool.steelAxe.name=钢斧
+item.thermalfoundation.tool.steelBow.name=钢强化弓
+item.thermalfoundation.tool.steelFishingRod.name=钢钓鱼竿
+item.thermalfoundation.tool.steelHammer.name=钢锤
+item.thermalfoundation.tool.steelHoe.name=钢锄
+item.thermalfoundation.tool.steelPickaxe.name=钢镐
+item.thermalfoundation.tool.steelShears.name=钢剪刀
+item.thermalfoundation.tool.steelShield.name=钢盾
+item.thermalfoundation.tool.steelShovel.name=钢铲
```
Comment (3TUSK) [inline src/main/resources/assets/thermalfoundation/lang/zh_CN.lang]: 锹。
All usage of “铲” shall be changed as well. Matching vanilla.

**Review by 3TUSK — CHANGES_REQUESTED** [review]

**Comment by KingLemming** [comment]
I really wish people would PR vs 1.12 and not master...I need to get rid of the master branch

**Comment by 3TUSK** [comment]
There is a change targeting branch function iirc...

**Pull request closed without merging**